app/utils/elastic-query.js

```diff
@@ -28,12 +28,8 @@ function dateRangeFilter(field, start, end) {
  */
 function termsFilter(field, terms, all = true) {
     if (terms && terms.length) {
-        if (field === 'contributors') {
```

Comment:
Nope! `type` and `types` are now unanalyzed, and have no `exact`/`raw` field. I tested it after merging and it works just fine.